site/en/external/vendor.md

```diff
@@ -160,3 +160,41 @@ is pinned in the VENDOR.bazel file. If a user does change the vendored source
 without pinning the repo, the changed vendored source will be used, but it will
 be overwritten if its existing marker file is
 outdated and the repo is vendored again.
+
+### Vendor registry files {:#vendor-registry-files}
+
+Bazel has to perform the Bazel module resolution in order to fetch external dependencies, which may require accessing registry files through internet.
+To achieve offline build, Bazel also vendors all registry files fetched from network under the `<vendor_dir>/_registries` directory.
+
+
+### Vendor symlinks {:#vendor-symlinks}
+
+External repositories may contain symlinks pointing to other files or directories.
+To make sure symlinks work correctly, Bazel uses the following strategy to rewrite symlinks in the vendored source:
+
+  - Create a symlink `<vendor_dir>/_bazel-external` that points to `$(bazel info output_base)/external`. It is refreshed by every Bazel command automatically.
```

Comment:
I would call the symlink `bazel-external` instead. It fits better with the existing symlinks and many projects have `bazel-*` in their `.gitignore` anyway. Since `bazel-external` isn't a valid canonical repo name with Bzlmod, conflicts wouldn't be possible.

Reply:
SG, done!